exampleSite/content/dev/blog/_global/sidebar-list.md

```diff
@@ -0,0 +1,7 @@
++++
```

Comment:
From a design perspective. The fragment in fragment has too much whitespace around it also it's currently not usable. Images shouldn't be there. Header text is too big etc.

That gives me a bit of a reason to think it's hard to optimize.

Reply:
The whitespace can be stripped out both by CSS or by extracting all the `.container`, `.container-fluid` and `.overlay` into a helper and call use the helper if the fragment is not in a slot. Other variables can be set to false if the fragment is registered as a slot or user can disable them manually.

Reply:
I think then a helper makes much more sense. As we want to have the option to have a stripped render mode for fragments or such.